README.md

```diff
@@ -108,36 +106,72 @@ Gambit has access to one of these meters and it is providing you a [live text fe
 98:17105
 99:3374
 100:17839
-```
 
-To help you on your way with data conversion I will give you a few clues based on the example data above:
+## Features
+- Display parsed data from the TUF-2000M ultrasonic energy meter.
+- User-friendly interface for easy data visualization.
+- Secure authentication for user access.
+
+## User Manual
+Information on the registries, variable names, units, notes, and formats are available on pages 39-42. docs/tuf-2000m.pdf
 
-- Register 21-22, Negative energy accumulator is -56.
-- Register 33-34, Temperature #1/inlet is 7.101173400878906.
-- Register 92, Signal Quality is 38.
+## Getting Started
+Follow these steps to set up the project and run the app locally:
 
-The registers and their respective datatypes are explained in detail in [docs/tuf-2000m.pdf](https://github.com/gambit-labs/tuf-2000m/blob/master/docs/tuf-2000m.pdf) on pages 39-42.
+1. Clone the repository to your local machine.
+2. Open the project in Android Studio.
+3. Build and run the app on an Android emulator or a physical device.
 
-## Your task
+## Usage
+1. Launch the app.
+2. Tap on the Sign-up text to register or use dummy credentials to log in (test@gmail.com / Atea@123#)
+3. View the parsed data from the TUF-2000M ultrasonic energy meter in the home view pager screen.
 
-Select one task from below. There is no single solution to these problems and we don't expect a complete solution to consider you for a position.
+## Technologies Used
+- Kotlin
+- Android Studio
 
-### Option 1: Create an REST API that parses the data to a readable form
-- .NET Core or maybe node.js?
-- JSON output is greatly appreciated
-- Easily deployable and testable
-- Authentication
+## Dependencies
+- [Retrofit](https://square.github.io/retrofit/) for making HTTP requests.
+- [Gson](https://github.com/google/gson) for JSON serialization and deserialization.
+- Hilt for dependency injection
+- Jetpack components
+- Firebase
+- -Etc...
 
-### Option 2: Create UI for showing the data for user
-- Modern javascript frameworks or maybe an mobile app. It's up to you
-- Easily deployable and testable
-- UX will be also a key factor on this
-- Authentication
+## Testing
+1. Signed apk distributed link for internal testing:  https://appdistribution.firebase.dev/i/0bde28dffa675f6a 
```

Comment:
To install the signed APK for testing or the below is the link of a video demo as well to know about the application